pkg/wrapper/helm/install.go

```diff
@@ -159,6 +155,13 @@ func (h *Helm) runInstall(args []string, vals map[string]interface{}, client *ac
 			} else {
 				return nil, err
 			}
```

Comment:
This else clause must be removed, as it is actually the `err != nil` part of the `action.CheckDependencies(...)` call, which is already handled below.